decidim-admin/app/views/decidim/admin/organization/_form.html.erb

```diff
@@ -74,6 +74,28 @@
 
 <div class="row">
   <div class="columns xlarge-6">
+    <%= form.file_field :official_img_header %>
+    <% if current_organization.official_img_header.present? %>
+      <label><%= t('.current_image') %></label>
+      <%= image_tag current_organization.official_img_header.url %>
```

Comment:
Any resized version to avoid using the original, that might potentially be huge.

* Homepage: "big"
* Logo: "medium"
* Favicon: "big"
* official_img_version and official_img_footer: original, although we should probably be storing the original versions and providing resized versions.